release_guide.md

```diff
@@ -0,0 +1,194 @@
+# Release Guide
+
+TBD
+
```

Comment:
What this TBD meaning?

Reply:
To be done.
This pr is just about Chinese version.

Reply:
OK

Reply:
To be done or To be discuss